From: Lars Ellenberg <lars.ellenberg@linbit.com>
Date: Thu, 3 Nov 2011 09:40:43 +0000 (+0100)
Subject: build: remove old sanity check to allow build against linux 3.1
X-Git-Url: http://git.drbd.org/gitweb.cgi?p=drbd-8.3.git;a=commitdiff_plain;h=790c26b09519b9e02b3a7cf7897fb17c2ae339bf

build: remove old sanity check to allow build against linux 3.1
---

diff --git a/drbd/Makefile b/drbd/Makefile
index 0985340..7a09603 100644
--- a/drbd/Makefile
+++ b/drbd/Makefile
@@ -90,11 +90,6 @@ else
     endif
   endif
 
-  KDIR_Makefile_PATCHLEVEL = $(shell test -e $(KDIR)/Makefile && grep "^PATCHLEVEL = " $(KDIR)/Makefile | cut -d " " -f 3)
-  ifneq ($(findstring $(KDIR_Makefile_PATCHLEVEL),12345),)
-    $(error "won't compile with this kernel version")
-  endif
-
   .PHONY: drbd.o default all greeting clean kbuild install dep tags
 
   drbd.o: greeting kbuild
